C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Point2D.cs

```diff
@@ -6,35 +6,49 @@
 
 namespace PE11VectorDistanceCalculation
 {
+    //Comments in this class apply for both objects
+    //Why have two objects? You can accomplish everything with one object and an extra property z here
+    //if you remove the methods from this class it can become a struct
     class Point2D : IEquatable<Point2D>
     {
+        //I like this class being immutable but no reason for the private set
         public int X { get; private set; }
         public int Y { get; private set; }
-
         public Point2D()
         {
+            //two problems with assigning X, Y here. 
+            //1. theyre value types, what is the default for the value types?
+            //2. what does this constructor accomplish?
             X = 0;
             Y = 0;
         }
 
+        //How do I set this object?
+        //Both constructors have internal assignments and no way for you to set the vector
+        //but I may not understand the assignment
```

Comment:
X and Y values are set via the constructor in the 2DCollectionArray using the random class.  I changed the default values, and it seems to work...